projects/plugins/jetpack/extensions/blocks/ai-assistant/ai-control.js

```diff
@@ -170,54 +190,68 @@ const AIControl = forwardRef(
 					/>
 
 					<div className="jetpack-ai-assistant__controls">
-						{ ! isWaitingState ? (
-							<Button
-								className="jetpack-ai-assistant__prompt_button"
-								onClick={ () => handleGetSuggestion( 'userPrompt' ) }
-								isSmall={ true }
-								disabled={
-									! userPrompt?.length || ! connected || siteRequireUpgrade || requireUpgrade
-								}
-								label={ __( 'Send request', 'jetpack' ) }
-							>
-								<Icon icon={ origamiPlane } />
-								{ ! isSm && __( 'Send', 'jetpack' ) }
-							</Button>
-						) : (
-							<Button
-								className="jetpack-ai-assistant__prompt_button"
-								onClick={ handleStopSuggestion }
-								isSmall={ true }
-								label={ __( 'Stop request', 'jetpack' ) }
-							>
-								<Icon icon={ closeSmall } />
-								{ __( 'Stop', 'jetpack' ) }
-							</Button>
-						) }
-
-						{ contentIsLoaded &&
-							! isWaitingState &&
-							( isInBlockEditor && promptType === 'generateTitle' ? (
+						<div className="jetpack-ai-assistant__prompt_button_wrapper">
```

Comment:
The diff got crazy here, but no change related with the buttons, just wrapped it in a new `div` to add the `keyboard` hint.